src/LEGO.AsyncAPI.Readers/YamlConverter.cs

```diff
@@ -47,16 +47,16 @@ public static JsonNode ToJsonNode(this YamlNode yaml)
             };
         }
 
-        private static JsonValue ToJsonValue(this YamlScalarNode yaml)
+        public static JsonValue ToJsonValue(this YamlScalarNode yaml)
```

Comment:
Why was this made public?

Reply:
I was testing just this function and every other function was was public so I figured make it consistent. I can change back

Reply:
Some of it, is public due to the Bindings project needing it for certain things, some things have been specifically copied to that project as well, but its a hassle keeping things up synced across.
but generally speaking, id like to keep things as private as possible.